commit 63d3dfc5cef9322f4e14f05f6a5bf71d7cac9145 (refs/changes/04/1704/3)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2017-04-05T10:41:14+00:00 (2 years, 6 months ago)
    
    TOOLS-1726 sdcadm create should not request usage of '--force-data-path'

diff --git a/lib/sdcadm.js b/lib/sdcadm.js
index 90ae0dc..3b99fbd 100644
--- a/lib/sdcadm.js
+++ b/lib/sdcadm.js
@@ -3635,6 +3635,10 @@ SdcAdm.prototype.genUpdatePlan = function genUpdatePlan(options, cb) {
 
             for (var i = 0; i < changes.length; i++) {
                 var ch = changes[i];
+                if (ch.type !== 'update-instance' &&
+                    ch.type !== 'update-service') {
+                    continue;
+                }
                 if (ch.service && dataPath.indexOf(ch.service.name) !== -1 &&
                     !options.forceDataPath)
                 {
